### DISCUSSION TITLE
Drag items OUT of <Reactflow />

### DISCUSSION THREAD

**Discussion by cfv1984** [xyflow/xyflow#3855] [Q&A]
I'd like to drop nodes onto an external div with an `onDrop={}` handler in it but nothing in the documentation explains how this would be possible. Has anyone had any luck with that? How did you do it?

**Comment by cfv1984** [comment]
Yeah so my client has decided we're dropping ReactFlow because of this. 

Just thought you might be interested. 

Cheers!

**Reply by moklick** [reply]
Would be interesting to understand the use case better. Maybe there is a solution for this.

**Reply by cfv1984** [reply]
Hi @moklick! 

The _general_ problem with what we're trying to do is that `<ReactFlow />` eats all the drag events of all its children. The specific use case we have in mind is like so:

Client has a bunch of node-based graphs he wanted rendered, describing a process. 

Nodes in this charted process need to be dropped on a `<div />` elsewhere on the page, controlled by a different team, causing a series of actions the client's client needs.

The page structure is basically like so:

```jsx
<Page>
  <UnrelatedUI />

  <main>
 
   <section>
     {/*this is where I am supposed to be able to drop the nodes*/}
    <FollowupTeamBin onDrop={handleProcessFollowup} />
    </section>

    <section>
    <MyTeamProcesses>

      {/* three components deep, there is a <ReactFlow /> container describing each separate process with its own rules */}
      <MyTeamInternalProcessViewer>

         <UnrelatedUI />

         <section>
           <UnrelatedUI />

           <section>
              {/* here */}
             <ReactFlowContainer />
           </section>

         </section>
      </MyTeamInternalProcessViewer>
    </MyTeamProcesses>

    </section>

  <main>

  <UnrelatedUI />

</Page>
```

If ReactFlow node wrappers eat all drag and drop events within its children, I simply cannot do drag and drop using this library; not with dnd-kit, nor with mere HTML5 drag and drop handlers.  This would make the library unsuitable for our purposes, which I'd hate because of the amount of rework this would cause..

An ideal solution for me would be to allow for a handle element that remains uncontrolled such that we can tell users "drag from here to toss the node on this bin", but _any_ strategy allowing for even rudimentary HTML5 drag and drop of nodes would be useful.

Any hint at a possible solution will be greatly appreciated.

Thanks!
